packages/backend-core/src/Base.ts

```diff
@@ -88,14 +90,13 @@ export class Base {
   };
 
   /**
-   *
-   * @param {string} token Clerk JWT verification token
+   * @param jwt Clerk JWT verification token
    * Modify the RSA public key from the Clerk PEM supplied and return a contructed CryptoKey.
    * You will find that at your application dashboard (https://dashboard.clerk.dev) under Settings ->  API keys
    *
    */
-  loadCryptoKey = async (key?: string): Promise<CryptoKey> => {
-    if (!key) {
+  loadCryptoKey = async (jwt?: string): Promise<CryptoKey> => {
+    if (!jwt) {
```

Comment:
This is actually a key here, how did the change came about ?

Reply:
Probably from the comment - but it should be `key`!